### ISSUE TITLE
🆕 Feature: Add Plugin `onload` timing

### ISSUE THREAD

**Issue by WuglyakBolgoink** [duddu/cordova-plugin-antitampering#5]
I think, plugin should be start with App
 
```xml
<feature name="AntiTamperingPlugin">
    <param name="ios-package" value="AntiTamperingPlugin" />
</feature>
```

with this option
```xml
<param name="onload" value="true" />
```

See: 

- **Android**: https://cordova.apache.org/docs/en/latest/guide/platforms/android/plugin.html#plugin-initialization-and-lifetime
- **iOS**: https://cordova.apache.org/docs/en/latest/guide/platforms/ios/plugin.html#plugin-initialization-and-lifetime

**Comment by duddu** [comment]
You are right, thanks!

**Comment by duddu** [comment]
Fixed with release 0.1.1
